### PR TITLE
Fix get rum data

### PR DESCRIPTION
**Pull request by nathanmarks**
### What does this PR do?
This fixes `getRumData()`, before it raised an exception, and also used the wrong format for the trace id

### Motivation
Was trying to get correlation working with RUM for the initial load

### Plugin Checklist

- [ ] Unit tests.
- [ ] TypeScript [definitions][1].
- [ ] TypeScript [tests][2].
- [ ] API [documentation][3].
- [ ] CircleCI [jobs/workflows][4].
- [ ] Plugin is [exported][5].

[1]: https://github.com/DataDog/dd-trace-js/blob/master/index.d.ts
[2]: https://github.com/DataDog/dd-trace-js/blob/master/docs/test.ts
[3]: https://github.com/DataDog/dd-trace-js/blob/master/docs/API.md
[4]: https://github.com/DataDog/dd-trace-js/blob/master/.circleci/config.yml
[5]: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/plugins/index.js

### CODE REVIEW

File: packages/dd-trace/src/opentracing/tracer.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ class DatadogTracer extends Tracer {
     this._url = this._exporter._url
     this._sampler = new Sampler(config.sampleRate)
     this._peers = config.experimental.peers
+    this._enableGetRumData = config.experimental.enableGetRumData
```
Comment (rochdev) [inline packages/dd-trace/src/opentracing/tracer.js:45]: The feature is still experimental right now so this cannot be added for the moment.

File: packages/dd-trace/src/proxy.js [hunk 2]
```diff
@@ -125,15 +125,7 @@ class Tracer extends BaseTracer {
...
-    if (!this._tracer._config.experimental.enableGetRumData) {
-      return ''
-    }
-    const span = this.scope().active().context()
-    const traceId = span._traceId
-    const traceTime = Date.now()
-    return `\
-<meta name="dd-trace-id" content="${traceId}" />\
-<meta name="dd-trace-time" content="${traceTime}" />`
+    return this._tracer.getRumData.apply(this._tracer, arguments)
```
Comment (rochdev) [inline packages/dd-trace/src/proxy.js:128]: If this is moved to the actual tracer it should also be added to the no-op implementation, which would always return an empty string.

**Review by rochdev — CHANGES_REQUESTED** [review]

File: packages/dd-trace/src/opentracing/tracer.js (hunk 1 above)
Comment (nathanmarks) [reply packages/dd-trace/src/opentracing/tracer.js:45]: @rochdev can we not use it to at least try it out? I understand it's a beta/experimental feature.

I needed to save the config somewhere, the initial config object is not saved anywhere.

The current implementation is completely bugged and the tests don't actually work, it's a false positive on the test sweet. They perform expectations in a callback without using `done()`.

This is what happens today due to how the method is attempting to read the config: https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/proxy.js#L128

```
TypeError: Cannot read property 'experimental' of undefined
```

We need to store this config value somewhere or it cannot be accessed.

Comment (rochdev) [reply packages/dd-trace/src/opentracing/tracer.js:45]: Yes, sorry I misread this as being in `config.js`. It's perfectly fine to set it on the tracer like this.

File: packages/dd-trace/src/proxy.js (hunk 2 above)
Comment (nathanmarks) [reply packages/dd-trace/src/proxy.js:128]: done

**Review by rochdev — APPROVED** [review]

**Comment by rochdev** [comment]
LGTM. Can you share was the exception was however for future reference?

**Comment by nathanmarks** [comment]
@rochdev explained in here https://github.com/DataDog/dd-trace-js/pull/1121#discussion_r508545500

**Pull request merged**